modules/rostests/apitests/shell32/CMakeLists.txt

```diff
@@ -67,7 +68,7 @@ add_rostests_file(TARGET shell32_apitest)
 
 # shell32_apitest_sub.exe
 add_executable(shell32_apitest_sub shell32_apitest_sub.cpp)
-target_link_libraries(shell32_apitest_sub cpprt atl_classes)
+target_link_libraries(shell32_apitest_sub cpprt cppstl atl_classes)
```

Comment:
A strong PR, I do value its intention very much.
Would it be possible to put that test into its own testsuite, so that the rest of the shell32apitest would not depend on that link-library? Or also possible: Is it an option to get this test implemented without using the cppstl?

Reply:
Had to add cppstl for things like C++ maps and sets. Would you suggest any idea where to add these? Or create a new module just for that?

Reply:
I do think a new test would be the wisest thing most likely, the idea is that this would allow me to use the existing and also very valuable shell32_apitests still on the older branches. But I just checked, the cpprt next to your added cppstl was also not present yet in 0.4.14. So it might be too late for my suggested change already.

Reply:
@JoachimHenze I've moved the new test into a new module, please re-review.

Reply:
Thank you!